packages/authorization-server/src/model/domain/models.ts

```diff
@@ -1,5 +1,24 @@
+import { IncomingHttpHeaders } from "http";
+import { authorizationServerApi } from "pagopa-interop-api-clients";
+import { z } from "zod";
+
 export interface InteropTokenResponse {
   access_token: string;
   token_type: string;
   expires_in: number;
 }
+
+export type TokenRequest = {
+  headers: IncomingHttpHeaders & { DPoP?: string };
+  body: authorizationServerApi.AccessTokenRequest;
+};
+
+export const tokenType = {
+  bearer: "Bearer",
+  dpop: "DPoP",
+} as const;
+export const TokenType = z.enum([
+  Object.values(tokenType)[0],
+  ...Object.values(tokenType).slice(1),
+]);
+export type TokenType = z.infer<typeof TokenType>;
```

Comment:
`TokenType` is already defined in the spec, making its declaration here redundant